eclair-core/src/main/scala/fr/acinq/eclair/payment/Relayer.scala

```diff
@@ -48,7 +48,7 @@ case class ForwardFail(fail: UpdateFailHtlc, to: Origin, htlc: UpdateAddHtlc) ex
 case class ForwardFailMalformed(fail: UpdateFailMalformedHtlc, to: Origin, htlc: UpdateAddHtlc) extends ForwardMessage
 
 case object GetUsableBalances
-case class UsableBalances(canSendMsat: Long, canReceiveMsat: Long, isPublic: Boolean)
+case class UsableBalances(canSendMsat: Long, canReceiveMsat: Long, shortChannelId: ShortChannelId, remoteNodeId: PublicKey, isPublic: Boolean)
```

Comment:
Reordering parameters would make more sense and be consistent with other API objects.

```suggestion
case class UsableBalances(remoteNodeId: PublicKey, shortChannelId: ShortChannelId, canSendMsat: Long, canReceiveMsat: Long, isPublic: Boolean)
```